post_installation_configuration/cluster-tasks.adoc

```diff
@@ -544,6 +544,21 @@ include::modules/machineset-delete-policy.adoc[leveloffset=+2]
 
 include::modules/nodes-scheduler-node-selectors-cluster.adoc[leveloffset=+2]
 
+include::modules/installation-extend-edge-nodes-aws-local-zones.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/installing_aws/installing-aws-localzone.html[Installing a cluster using AWS Local Zones]
+
+* xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc[Understanding taints and tolerations]
+
+* xref:../logging/config/cluster-logging-tolerations.adoc[Using tolerations to control OpenShift Logging pod placement]
+
+.Next steps
```

Comment:
It makes more sense to me for the Additional Resources section to be after the Next Steps section. I see it after the Procedures or Verifications in other parts of the docs as well, but I don't see another example with the Additional Resources and Next Steps. WDYT?

Reply:
Yes, I agree. The next steps and additional resources section still relate to my content additions, so no confusion would be caused with rearranging the IDs.